globus_portal_framework/settings.py

```diff
@@ -182,7 +182,7 @@ def get_rfm(search_result):
 
 ROOT_URLCONF = 'globus_portal_framework.urls'
 
-BASE_TEMPLATES = 'globus-portal-framework/v2/'
+BASE_TEMPLATES = 'globus-portal-framework/v3/'
```

Comment:
Could we leave these as v2 for now? New folks can switch to v3 manually for v0.4.x, and once we're ready to roll out v0.5.x we can make this default.

Reply:
Sounds good!